src/Kunstmaan/GeneratorBundle/Command/InstallCommand.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+namespace Kunstmaan\GeneratorBundle\Command;
+
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Question\Question;
+use Symfony\Component\Console\Output\NullOutput;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputDefinition;
+use Symfony\Component\Console\Output\OutputInterface;
+use Sensio\Bundle\GeneratorBundle\Command\Validators;
+use Symfony\Component\Console\Question\ChoiceQuestion;
+use Symfony\Component\Process\Exception\RuntimeException;
+use Sensio\Bundle\GeneratorBundle\Command\Helper\QuestionHelper;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Debug\Exception\ContextErrorException;
+
+class InstallCommand extends ContainerAwareCommand
```

Comment:
For new commands, we don't extend the ContainerAwareCommand anymore. It should work with service injections.